### DISCUSSION TITLE
[Question] Transferring Outfits

### DISCUSSION THREAD

**Discussion by WiiBii2** [Monika-After-Story/MonikaModDev#8323] [QA - tech support + general] [answered]
Hello! I would like to copy and paste Monika's outfits from my current drive to a second installation of MAS. Should I copy and paste everything from the `mod_assets` folder, or some other folder? Or should I just manually add the files the way they were supposed to be added? Thanks in advance!

**Comment by lunulae** [comment, marked as answer]
If you're only transferring outfits, then `mod_assets` should work. If you want to get really specific you'll just need the `monika` and `thumbs` folders.
